content/en/blog/the-time-is-now/index.md

```diff
@@ -0,0 +1,100 @@
+How can we utilize this technology in a broader range of use cases, for instance,
+inter-organizational transactions?
+
+Therefore, we need more real-world use cases implemented with SSI technology to develop it correctly.
+After all, the use cases should ultimately be the source of the requirements, the drivers of how
+to bring the technology further. And if use case implementors are waiting for the technology
+to become ready, we have a severe chicken-egg problem.
+
+Instead of overthinking the low-level crypto operations, etc., the teams should concentrate
+on the high-level building blocks of SSI. The starting point should be clarifying
+the issuer's, holder's, and verifier's roles. What exactly happens when data is
+issued and verified? And how can the team apply the verifiable data in the use case
+in question: who are the issuer, holder, and verifier? Furthermore, what data does
+the issuer sign, and which part is available to the verifier?
+
+## It Is All About Communication
+
+After figuring out answers to these questions, the selection of the technical stack
+becomes more relevant. However, even then, I would put less weight on the credential
+format selection. What is even more important is the used credential exchange protocol.
+
+For instance, if your primary use case is user identification and authentication,
+and you wish to create an architecture that suits well and fast in the legacy world,
+the most straightforward choice is a protocol offering strict client-server roles
+and HTTP-request-response style API. In this style, the issuer, holder, and
+verifier roles are hard-coded, and one party cannot simultaneously play multiple parts.
+
+{{< imgproc cover Fit "925x925" >}}
+<em>DIDComm and Hyperledger Aries are examples of a symmetric protocols. Findy Agency
+supports currently both.</em>
+{{< /imgproc >}}
+
+However, choosing a symmetric protocol may widen the spectrum of your use cases.
+Participants can execute different functions during the interaction in a symmetric protocol.
+So, for example, in the user authentication scenario, in addition to the service (server)
+authenticating the user (client) based on verified data, the individual could
+also get a verification from the service-operating organization of its authenticity.
+
+## Integrate Carefully
+
+Whichever stack you choose, there are two things to keep in mind
+
+1. Implement the integration to the library or service providing SSI functionality following
+well-known modular principles so that it is easy to replace with a different library or service.
+1. Do not let the underlying technology details pollute your application code.
+Thus, ensure the SSI tooling you use hides the implementation details related
+to credentials formats and protocol messages from your application.
+This approach ensures that changes, for example, to the credential data format,
+have a manageable effect on your application.
+
```

Comment:
Discussion comment: I dare to suggest that the need for network layer protocol is quite apparent when reading this list. I think it would be a good idea to talk about it further.